engine/src/main/java/org/terasology/engine/world/block/shapes/BlockMeshPart.java

```diff
@@ -80,32 +81,38 @@ public BlockMeshPart mapTexCoords(Vector2f offset, float width, int frames) {
         return new BlockMeshPart(vertices, normals, newTexCoords, indices, frames);
     }
 
-    public void appendTo(ChunkMesh chunk, ChunkView chunkView, int offsetX, int offsetY, int offsetZ,
-                         ChunkMesh.RenderType renderType, Colorc colorOffset, ChunkVertexFlag flags) {
```

Comment:
I don't think we need to pass in the RenderType. if we still want the enumeration for the type we can expose that as a free function.